### PR TITLE
[IIGO] [1/3] Analysis and evaluation

### PR DESCRIPTION
**Pull request by Spruudis**
For reference see book chapters 6.4.3 (Institutional Analysis and Development framework),
6.4.2 (Ostrom), 6.6.1(Zone of Dignity)

Also redid some rules because I realised an "if and only if" statement does not imply permission

### CODE REVIEW

File: 05_iigo/index.tex [hunk 1]
```diff
@@ -948,7 +948,64 @@ \subsection{Rule Representation}
...
+\section{Analysis and Reflection}
+\subsection{Ostrom’s Design Principles and IIGO}
+\subsubsection{Institutional Analysis and Development framework overview of IIGO}
+
+In this section, we start the evaluation of IIGO by briefly stating or re-visiting the institution's \emph{Bio-Physical Characteristics}, \emph{Attributes of the Community}, \emph{Rules-in-Use}, as well as defining the distinct \emph{Action Arenas} and relevant \emph{Actors}.
+
+\begin{itemize}
+    \item Bio-Physical Characteristics
+    
+    Resources are stored in two distinct \emph{facilities}: in private pools and in the common pool. The only \emph{facility} that can generate resources is foraging. Resources are expended through the cost of living, disaster events, cost of IIGO actions and potentially foraging. In the scope of this coursework, IIGO is only concerned with managing the movement of resources between private pools and the common pool.
```
Comment (yp717) [inline 05_iigo/index.tex:960]: I would bold the item titles here and put a colon or the formatting can be a bit unpredictable

File: 05_iigo/index.tex [hunk 2]
```diff
@@ -948,7 +948,64 @@ \subsection{Rule Representation}
...
+        
+        \item The third decision arena of IIGO is that surrounding sanctions. Here the Judge is the only actor involved in the decision making of whether to and how harshly to sanction islands, as well as whether to pardon islands.
+        
+        \item The fourth decision arena of IIGO is that surrounding monitoring of role powers and elections. The two are interlinked whereby the elected actors have to participate in not only the decision making surrounding elections but also the decision making in monitoring, which in turn can alter the rules in play to oblige other actors to hold elections. Moreover, elections involve the involvement of all islands voting, thus providing information to the island responsible for the appointment of a role. It is important to note is the repetitive nature of decisions roles have to make regarding elections, followed by decisions regarding monitoring, followed by elections, e.t.c.
```
Comment (yp717) [inline 05_iigo/index.tex]: minor but etcetera and is abbreviated by etc. - not sure what e.t.c stands for - quick fix

**Review by yp717 — CHANGES_REQUESTED** [review]
one minor suggestion + one mis-abbreviated word 

checked the logic around Ostrom and it seems to make sense to be so happy to merge after those two quick changes

File: 05_iigo/index.tex (hunk 2 above)
Comment (Spruudis) [reply 05_iigo/index.tex]: I wrote that late at night and kept staring at it thinking it doesn't look right but I just wrote off as 1AM madness haha

File: 05_iigo/index.tex [hunk 3]
```diff
@@ -948,7 +948,64 @@ \subsection{Rule Representation}
...
+
+    Rule modification is the result of the voting system. If all the available rules governing this process are followed by the agents, all agents equally contribute to this decision.  
+
+    Of note here is the Speaker's power to decide which islands are to participate in the vote. This power is governed by Rule~\ref*{rule:all_islands_vote} and is a remnant of a system of sanctions we considered during design, which could bar islands from participating in a vote. It is the Speakers power to call a vote, i.e. send a communication message to all the agents, hence it would have been the Speakers power to enforce such sanctions, by choosing which islands to send a vote request to. This system is not implemented, which begs for such a decision to be more regulated through a constitutional rule.
+
+    The enforcement of collective choice rules is covered by Ostrom's principles 4 and 8
```
Comment (ezgimez) [inline 05_iigo/index.tex]:
```suggestion
    The enforcement of collective choice rules is covered by Ostrom's principles 4 and 8.
```

**Review by yp717 — APPROVED** [review]
lgtm now

**Review by ezgimez — APPROVED** [review]
approving without reading full, due to time constraints

**Pull request merged**